### DISCUSSION TITLE
版本更新好快啊，有没有更新日志呢。

### DISCUSSION THREAD

**Discussion by roy1866** [Akifyss/obsidian-border#268] [General]
版本更新好快啊，有更新日志么。想知道更新了哪些内容。^_^

**Comment by Akifyss** [comment]
可以在这里看到每次更新的信息，但其实都是一些小优化/修复，也不用太在意。
https://github.com/Akifyss/obsidian-border/commits/main

**Comment by roy1866** [comment]
> 可以在这里看到每次更新的信息，但其实都是一些小优化/修复，也不用太在意。 https://github.com/Akifyss/obsidian-border/commits/main
感谢，相当完美的主题~
